packages/Molecules/List/index.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div
+    class="rpl-list"
+    :class="{
+      'rpl-list--normal': (size === 'normal'),
+      'rpl-list--large': (size === 'large')
+    }"
+  >
+    <h2 v-if="title" class="rpl-list__title">{{ title }}</h2>
+    <div v-if="list" class="rpl-list__list">
+      <div v-for="(item, index) in list" :key="index" class="rpl-list__list-item">
+        <span class="rpl-list__icon">
+          <rpl-icon v-if="item.symbol" :symbol="item.symbol" :color="item.color || iconColor" :size="iconSize(item)" /></span>
+          <rpl-link v-if="item.link" :href="item.link" class="rpl-list__text">{{ item.text }}</rpl-link>
+          <span v-else :href="item.link" class="rpl-list__text">{{ item.text }}</span>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import RplLink from '@dpc-sdp/ripple-link'
+import RplIcon from '@dpc-sdp/ripple-icon'
+
+export default {
+  name: 'RplList',
+  props: {
+    title: String,
+    size: { type: String, default: 'normal' },
+    iconScale: { type: Number, default: 1 },
+    iconColor: { type: String, default: 'primary' },
+    list: Array
+  },
+  components: {
+    RplLink,
+    RplIcon
+  },
+  methods: {
+    iconSize (item) {
+      return (item.size ? item.size : 1) * this.iconScale
```

Comment:
An icon size can be set per icon. Icons can also be globally scaled using iconScale on the component.
If an icon size is set to 1.2, then iconScale is set to 2.5, the resulting icon will be (1 * 1.2 * 2.5) = 3.
This means minor adjustments to icon sizes can be made on a per icon basis, then uniformly scaled up or down as needed.